ui/arduino/store.js

```diff
@@ -166,7 +166,7 @@ function store(state, emitter) {
     let editor = state.cache(AceEditor, 'editor').editor
     let contents = cleanCharacters(editor.getValue())
     editor.setValue(contents)
-    let filename = state.selectedFile || 'undefined'
+    let filename = cleanCharacters(state.selectedFile) || 'undefined'
```

Comment:
```suggestion
    let filename = cleanCharacters(state.selectedFile || 'undefined')
```

This should handle when `state.selectedFile` is `null`.